src/config/dynamic-theme-fixes.config

```diff
@@ -3902,6 +3902,38 @@ INVERT
 
 ================================
 
+jumia.com
+jumia.co
+jumia.ma
+jumia.ci
+jumia.dz
+jumia.ug
+jumia.sn
+zando.co.za
+
+INVERT
+svg.ic[role="img"]:not([aria-label="Zando logo"])
+.logo > a > img
+.inbox > a:not(:nth-of-type(1)) > svg
+img[src*="jumia-group-logo.png"]
+img[src*="jumia_logo_small_checkout.png"]
+img[src*="/Jumia-Pay"]
+li.logo:nth-child(1) > .-i-jumia-logo
+img[src*="empty-cart.png"]
+.-header > img:nth-child(1)
+.-gy5 > .-fs0 > .vent-link[title*="Rewards"]
+.-gy5 > .-fs0 > .vent-link[title*="Pay"]
+.-gy5 > .-fs0 > .vent-link[title*="Food"]
+.-gy5 > .-fs0 > .vent-link[title*="Party"]
+.-gy5 > .-fs0 > .vent-link[title*="Now"]
```

Comment:
Small icons at the top
![Small icons at the top](https://user-images.githubusercontent.com/14346945/100476878-65e8ba80-30ef-11eb-9d30-ba278fc5eec4.jpg)
![Small icons at the top (2)](https://user-images.githubusercontent.com/14346945/100476881-6719e780-30ef-11eb-8334-4363a5b5d111.jpg)